packages/blade/src/tokens/theme/overrideTheme.ts

```diff
@@ -47,7 +47,12 @@ type OverrideTheme = {
  */
 const overrideTheme = ({ baseThemeTokens, overrides }: OverrideTheme): ThemeTokens => {
   if (__DEV__) {
-    if (!isEqual(baseThemeTokens, paymentTheme) && !isEqual(baseThemeTokens, bankingTheme)) {
+    if (
+      !hasSameObjectStructure(
```

Comment:
Keeping this. Without this overriding a theme generated by `createTheme` won't be possible if the use-case ever comes up for us.